package.json

```diff
@@ -14,7 +14,7 @@
     "prebuild:dist": "rm -rf dist/*",
     "build:dist": "rollup -c && rollup -c --environment PRODUCTION",
     "build:watch": "npm run build:lib -- --watch",
-    "test": "mocha \"./src/**/*.test.js\" --require babel-core/register --timeout 5000",
```

Comment:
Could we keep running mocha in node as default and add `npm run test:browsers` for testing with browsers?